src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -927,6 +927,32 @@ class Customer2
         private int _value = 0;
 //]
 }}
+";
+
+        private static readonly string s_avoidUnusedMembers = $@"
+class Customer1
+{{
+//[
+        // {ServicesVSResources.Prefer_colon}
```

Comment:
this probably needs something indicating that all that cod ewas removed... not sure.  can you show how this looks?

Reply:
![image](https://user-images.githubusercontent.com/10605811/44618711-01fc2b00-a830-11e8-8c21-f5daef3d3fff.png)